deno_webgpu/render_pass.rs

```diff
@@ -195,14 +193,15 @@ pub fn op_webgpu_render_pass_execute_bundles(
 #[serde]
 pub fn op_webgpu_render_pass_end(
     state: &mut OpState,
+    #[smi] _command_encoder_rid: ResourceId,
```

Comment:
this parameter is unused in this function, but it is part of the denoland implementation of deno_webgpu, and it's expected to be present by `01_webgpu.js` in this repository

https://github.com/gfx-rs/wgpu/blob/61b7063d8e026415cd840d7097882c16304dc90b/deno_webgpu/01_webgpu.js#L3867-L3870